google/cloud/apigee/registry/applications/v1alpha1/registry_manifest.proto

```diff
@@ -0,0 +1,36 @@
+syntax = "proto3";
+
+package google.cloud.apigee.registry.applications.v1alpha1;
+
+option java_package = "com.google.cloud.apigee.registry.applications.v1alpha1";
+option java_multiple_files = true;
+option java_outer_classname = "RegistryManifestProto";
+option go_package = "github.com/apigee/registry/rpc;rpc";
+
+// Manifest represents the dependency relation between resources, which 
+// is used by the controller to keep the resources up-to-date.
+message Manifest {
+  // List of dependency relations.
+  repeated Resource resources = 1 [
+		(google.api.field_behavior) = REQUIRED
+	];
+}
+
+message Resource {
+	string pattern = 1 [
```

Comment:
I was wondering if it makes sense to add some validation annotation here to make sure the pattern is a valid query for the registry.

Reply:
Could we start with a comment showing the expected format? Like [this](https://github.com/apigee/registry/blob/21587ed876e8bd4073334eb8c24eba0fa2a9a871/google/cloud/apigee/registry/v1/registry_models.proto#L88)?

Reply:
Also, I agree that `pattern` is the right name for this. I briefly considered `name_pattern` after finding it [here](https://github.com/googleapis/googleapis/blob/7e91915c247a5c84ab5fcdf11ce2bdf7404f2fcc/google/cloud/binaryauthorization/v1beta1/resources.proto#L101), but that's an isolated instance and `pattern` is used throughout the AIPs and [here](https://github.com/googleapis/googleapis/blob/7cde5d0df08801e00fc45df1546942aa2692d5c3/google/api/resource.proto#L218), which seems more authoritative.

Reply:
Added some examples in the comment.